### PR TITLE
Adds nightfox variants

### PR DESCRIPTION
**Pull request by ascander**
Added all nightfox variants and updated existing themes from source.

### CODE REVIEW

**Comment by chrisduerr** [comment]
Why did you add all of them? Are you interested in all of them or just a specific one and you added the others for completeness?

**Comment by ascander** [comment]
Both; I regularly use the dusk|night|nord|tera variants, and decided to throw them all in for completeness.

File: themes/dayfox.toml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+cyan = "#22676d"
+white = "#cec6bf"
+
+[[colors.indexed_colors]]
+index = 16
+color = "#955f61"
+
+[[colors.indexed_colors]]
+index = 17
+color = "#a440b5"
```
Comment (chrisduerr) [inline themes/dayfox.toml:79]: Please remove the indexed colors from all themes.

**Review by chrisduerr — CHANGES_REQUESTED** [review]

File: themes/dayfox.toml (hunk 1 above)
Comment (ascander) [reply themes/dayfox.toml]: Done!

**Review by chrisduerr — APPROVED** [review]

**Pull request merged**